.github/workflows/provider-compatibility.yml

```diff
@@ -109,13 +107,9 @@ jobs:
           #     `tgz' is the name of the tarball use to keep the artifacts of
           #         the build.
           {
-            name: openssl-3.0,
-            dir: branch-3.0,
-            tgz: branch-3.0.tar.gz,
-          }, {
-            name: openssl-3.1,
-            dir: branch-3.1,
-            tgz: branch-3.1.tar.gz,
+            name: '',
+            dir: HEAD,
+            tgz: HEAD.tar.gz,
```

Comment:
Doesn't this change lose some of the coverage?
3.0 vs 3.1 e.g.

Reply:
This is just a testing commit to be dropped.

Reply:
Ah okay, it wasn't obvious.